.github/workflows/reusable-build-test-release.yml

```diff
@@ -4,7 +4,13 @@ on:
     inputs:
       marker:
```

Comment:
let's make it modinput_marker

Reply:
@artemrys  suggested "Let's avoid making this breaking change - let's not rename the existing marker."